macro-plantuml-macro/src/main/java/org/xwiki/contrib/plantuml/PlantUMLMacroParameters.java

```diff
@@ -42,6 +44,16 @@ public void setServer(String serverURL)
         this.serverURL = serverURL;
     }
 
+    /**
+     * @param format see {@link #getFormat()}. Valid values: svg, png, txt
+     *
```

Comment:
This can be improved. Best is to use: `Valid values are: {@code svg}, {@code png} and {@code txt}`, or link to some plantuml doc explaining the valid values.

Reply:
Also missing a `@since` (and ideally an `@Unstable` annotation too).